Loop/View Models/BolusEntryViewModel.swift

```diff
@@ -124,6 +125,16 @@ final class BolusEntryViewModel: ObservableObject {
             .store(in: &cancellables)
     }
 
+    private func observeRecommendedBolusChanges() {
```

Comment:
I took the approach of just subscribing to the recommended bolus changes, then automatically accepting them. This isn't ideal but the optimal solution would involve quite-significant refactoring